platform/ios/src/MGLMapView.mm

```diff
@@ -1714,6 +1717,12 @@ - (void)handlePanGesture:(UIPanGestureRecognizer *)pan
 {
     if ( ! self.isScrollEnabled) return;
 
+    if (!self.allowScrollGesturesDuringRotateOrZoom) {
```

Comment:
Is it possible to move this code to `gestureRecognizerShouldBegin`?

Reply:
Yes